pipelines/HyperSuprimeCam/ApPipeWithFakes.yaml

```diff
@@ -34,5 +34,3 @@ tasks:
       connections.fakesType: parameters.fakesType
       insertFakes.doSubSelectSources: True
       insertFakes.select_col: 'isVisitSource'
-      calibrate.photoCal.match.referenceSelection.magLimit.fluxField: i_flux
-      calibrate.photoCal.match.referenceSelection.magLimit.maximum: 22.0
```

Comment:
Please don't remove these, unless you add them to `config/HSC/processCcdWithFakes.py` instead

Reply:
Never mind, see above!